package.json

```diff
@@ -8,12 +8,23 @@
     "request": "^2.88.0"
   },
   "devDependencies": {
-    "eslint": "^5.5.0",
-    "jest": "^23.5.0"
+    "ajv": "^6.5.2",
+    "eslint": "^5.6.0",
+    "eslint-config-tamia": "^6.0.1",
+    "eslint-plugin-prettier": "^2.6.2",
+    "jest": "^23.6.0",
+    "memfs": "2.9.4",
+    "prettier": "^1.14.2"
+  },
+  "engines": {
+    "node": ">=8.6"
```

Comment:
are you really sure that this low version of Node is the minimal one? We are using features from Ecma2017.